src/plugins/csv/csvplugin.cpp

```diff
@@ -43,62 +45,64 @@ CsvPlugin::CsvPlugin()
 
 bool CsvPlugin::write(const Map *map, const QString &fileName)
 {
+    // Extract file name without extension and path
+    QFileInfo fileInfo(fileName);
+    fileInfo.setCaching(false);
```

Comment:
Ehm... I got confused with Qt docs. I wanted to prevent QFileInfo from accessing the actual filesystem (I just want QFileInfo to strip out parts of the path). I'll take line 50 out.